src/js/main.js

```diff
@@ -14,8 +14,8 @@ Site = {
 
     $(document).ready(function () {
       Site.Shapes.init();
-      Site.Map.init();
       Site.Minimap.init();
+      Site.Map.init();
```

Comment:
Cuz otherwise it wasn't working. I was thinking we should move the `window` object from `Site.Map` into just `Site`